PWGJE/DataModel/EMCALClusters.h

```diff
@@ -152,10 +152,18 @@ using EMCALClusterCell = EMCALClusterCells::iterator;
 using EMCALAmbiguousClusterCell = EMCALAmbiguousClusterCells::iterator;
 namespace emcalmatchedtrack
 {
-DECLARE_SOA_INDEX_COLUMN(Track, track); //! linked to Track table only for tracks that were matched
+DECLARE_SOA_INDEX_COLUMN(Track, track);                 //! linked to Track table only for tracks that were matched
+DECLARE_SOA_ARRAY_INDEX_COLUMN(McParticle, mcParticle); //! Array of MC particles that deposited energy in this calo cell
```

Comment:
where is this used?

Reply:
Ah good catch, that's some left over copy paste thing.
I will remove it with the next commit